src/main/java/org/folio/linked/data/validation/dto/LccnPatternValidator.java

```diff
@@ -1,21 +1,35 @@
 package org.folio.linked.data.validation.dto;
 
+import static java.lang.Boolean.TRUE;
 import static org.apache.commons.collections4.CollectionUtils.isEmpty;
 
 import jakarta.validation.ConstraintValidator;
 import jakarta.validation.ConstraintValidatorContext;
 import java.util.regex.Pattern;
+import lombok.RequiredArgsConstructor;
 import org.folio.linked.data.domain.dto.LccnRequest;
 import org.folio.linked.data.validation.LccnPatternConstraint;
+import org.folio.linked.data.validation.spec.SpecProvider;
+import org.folio.rspec.domain.dto.SpecificationRuleDto;
 
+@RequiredArgsConstructor
 public class LccnPatternValidator implements ConstraintValidator<LccnPatternConstraint, LccnRequest> {
 
   private static final Pattern LCCN_STRUCTURE_A_PATTERN = Pattern.compile("( {3}|[a-z][ |a-z]{2})\\d{8} ");
   private static final Pattern LCCN_STRUCTURE_B_PATTERN = Pattern.compile("( {2}|[a-z][ |a-z])\\d{10}");
 
+  private final SpecProvider specProvider;
+
   @Override
   public boolean isValid(LccnRequest lccnRequest, ConstraintValidatorContext constraintValidatorContext) {
-    if (isCurrent(lccnRequest)) {
+    var isEnabled = specProvider.getSpecRules()
+      .stream()
+      .filter(rule -> rule.getCode().equals("invalidLccnSubfieldValue"))
```

Comment:
Let us define a constant for "invalidLccnSubfieldValue"

Reply:
defined a constant for now